pro/src/components/AccessibilitySummarySection/AccessibilitySummarySection.tsx

```diff
@@ -1,63 +1,61 @@
 import React from 'react'
 
-import {
-  GetCollectiveOfferTemplateResponseModel,
-  GetCollectiveOfferResponseModel,
-  GetIndividualOfferResponseModel,
-} from 'apiClient/v1'
 import { SummaryDescriptionList } from 'components/SummaryLayout/SummaryDescriptionList'
 import { SummarySubSection } from 'components/SummaryLayout/SummarySubSection'
 import { AccessiblityEnum } from 'core/shared'
 import { AccessibilityLabel } from 'ui-kit/AccessibilityLabel'
 
 import styles from './AccessibilitySummarySection.module.scss'
 
+// This interface is verified by both collective/individual offers and venues
+interface AccessibleItem {
```

Comment:
détail: préférer `type Machin = {`

Reply:
The TS doc disagrees :slightly_smiling_face: 

![image](https://github.com/pass-culture/pass-culture-main/assets/6317823/1be9e614-799d-42e9-b3ce-8402231abff1)